packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -24,8 +25,8 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
   const template = (
     <div
       ref={popperRef}
-      id="ATL-Combobox-Content"
-      data-testid="ATL-Combobox-Content"
+      id={COMBOBOX_MENU_ID}
```

Comment:
ah. yeah I don't think there's any functional issues, at least I sure hope not since we have multiple comboboxes on a few pages now.

I was thinking more about testing, where if you use this selector it's going to match multiple. maybe that's a good thing though? consumers should try their very best to avoid testing Atlantis. a more valuable test would be to find a button that says "select thing" and then choose an option or something like that - rather than checking the internal ID to see if the combobox menu is showing up which is something we have unit/integration tests for already.